lib/src/main/resources/bag/metadata/afm/afm.xsd

```diff
@@ -75,7 +78,11 @@
             <xs:documentation xml:lang="dut">technische bestandsspecificatie</xs:documentation>
         </xs:annotation>
     </xs:element>
-
+    
+    <xs:element name="file_name" substitutionGroup="dcterms:title"/>
```

Comment:
So I should change https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/1d6812896df165ef2a5ae62d9912114d3b75eef6/src/main/scala/nl/knaw/dans/easy/fedoratobag/FileItem.scala#L88 ?

Reply:
no, the way you did it is better